packages/components/number-input/src/use-number-input.ts

```
@@ -0,0 +1,569 @@
+    labelProps,
+    inputProps,
+    incrementButtonProps,
+    decrementButtonProps,
+    descriptionProps,
+    errorMessageProps,
+    isInvalid: isAriaInvalid,
+    validationErrors,
+    validationDetails,
+  } = useAriaNumberInput(originalProps, state, domRef);
+
+  const inputValue = isNaN(state.numberValue) ? "" : state.numberValue;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)